docs/env-vars-operator.md

```diff
@@ -0,0 +1,191 @@
+You can set environment variables in the following ways:
+
+* For installations via `kubectl`, edit the Operator Deployment manifest (`deploy/operator.yaml`) before applying it, or modify the existing Deployment using `kubectl patch` or `kubectl edit`.
+* For Helm installations you can set environment variables through Helm values.
+* For installations on OpenShift, you can configure environment variables through the OLM subscription.
+
+## Available environment variables
+
+### `LOG_STRUCTURED`
+
+Controls whether Operator logs are structured (JSON format) or plain text. Available since Operator version 1.12.0
+
+| Value type | Default | Example |
+| ---------- | ------- | ------- |
+| string     | `"false"` | `"true"` |
+
+When set to `"true"`, the Operator outputs logs in structured JSON format, which is useful for log aggregation systems. When set to `"false"` (default), logs are in plain text format.
+
+**Example configuration:**
+
+```yaml
+env:
+ - name: LOG_STRUCTURED
+   value: "true"
+```
+  
+### `LOG_LEVEL`
+
+Sets the verbosity level of Operator logs. Available since Operator version 1.12.0
+
+| Value type | Default | Example |
+| ---------- | ------- | ------- |
+| string     | `"INFO"` | `"DEBUG"` |
+
+Valid values are:
+
+* `"DEBUG"` - Most verbose, includes detailed debugging information
+* `"INFO"` - Standard informational messages (default)
+* `"WARN"` - Warning messages only
+* `"ERROR"` - Error messages only
+
+**Example configuration:**
+
+```yaml
+env:
+ - name: LOG_LEVEL
+   value: "DEBUG"
+```
+
+### `WATCH_NAMESPACE`
+
+Specifies which namespaces the Operator should watch for Custom Resources.
+
+| Value type | Default | Example |
+| ---------- | ------- | ------- |
+| string     | Operator's namespace | `"pxc,pxc-dev"` or `""` |
+
+* If set to a comma-separated list of namespaces, the Operator watches only those namespaces (cluster-wide mode)
+* If set to an empty string (`""`), the Operator watches all namespaces in the cluster
+* If not set, the Operator watches only its own namespace
+
+**Example configuration for cluster-wide mode:**
+
+```yaml
+env:
+ - name: WATCH_NAMESPACE
+   value: "pxc,pxc-dev,pxc-prod"
+```
+
+See [Cluster-wide installation](cluster-wide.md) for more details.
+
+### `DISABLE_TELEMETRY`
+
+Disables the Operator's telemetry data collection.
+
+| Value type | Default | Example |
+| ---------- | ------- | ------- |
+| string     | `"false"` | `"true"` |
+
+When set to `"true"`, the Operator does not send anonymous telemetry data to Percona.
+
+**Example configuration:**
+
+```yaml
+env:
+ - name: DISABLE_TELEMETRY
+   value: "true"
+```
+
+See [Telemetry](telemetry.md) for more information about what data is collected.
+
+### `S3_WORKERS_LIMIT`
+
+This variable limits the number of parallel workers used for both backup deletion from the S3-compatible storage (AWS S3, MinIO, etc.). The same worker pool is used for both operations. Available since Operator version 1.8.0.
```

Comment:
```suggestion
This variable limits the number of parallel workers used for backup deletion from the S3-compatible storage (AWS S3, MinIO, etc.). Available since Operator version 1.8.0.
```